src/site/includes/header.html

```diff
@@ -34,6 +34,9 @@
     {% include "src/site/assets/js/record-event.js" %}
   </script>
 
+  <link rel="stylesheet" data-entry-name="web-components.css">
+  <script nonce="**CSP_NONCE**" data-entry-name="web-components.js"></script>
```

Comment:
This ensures our CSS and JS for the Web Components will be loaded onto every page.